DoxygenRule.cmake

```diff
@@ -88,6 +88,19 @@ if(NOT COMMON_ORGANIZATION_NAME)
   set(COMMON_ORGANIZATION_NAME Unknown)
 endif()
 
+# collect application help page
+get_property(__help GLOBAL PROPERTY ${PROJECT_NAME}_HELP)
```

Comment:
How about file(GLOB __help "${PROJECT_BINARY_DIR}/help/*.md")  instead of a global property? Potentially a bit more flexible and decoupled (for someone who wants to add his own help files in there).